### DISCUSSION TITLE
Integration of TensorBoard in DeepXDE.

### DISCUSSION THREAD

**Discussion by vassilismin** [lululxvi/deepxde#1577] [Q&A]
Hello everyone!
I'm wondering if it is possible to exploit the tensorboard for the deepxde models. Looking into the deepxde documentation, the callbacks do not contain the tensorboard callback. I also tested to use import it from tensorflow and add this ass callback in my deepxde model but it returned the following error

`AttributeError: 'Model' object has no attribute 'distribute_strategy'`.

So, is it possible to integrate Tensorboard? If yes do you have any tips to accomplish that?

**Comment by lululxvi** [comment]
I don't see why it is not possible to integrate Tensorboard. But I don't have much experience with TensorBoard.
